ImputationPipeline/PRSWrapper.wdl

```diff
@@ -75,10 +75,20 @@ workflow PRSWrapper {
       is_control_sample = is_control_sample
   }
 
+  Array[File] missing_sites_shifted = select_all(ScoringImputedDataset.missing_sites_shifted_scores)
+
+  call CombineMissingSitesShiftedScores {
+    input:
+      missing_sites_shifted = missing_sites_shifted,
+      lab_batch = lab_batch
```

Comment:
I have no idea how this works, because it shouldn't know `lab_batch` at this point. 
```suggestion
      lab_batch = lab_batch_id
```

Reply:
Looks like an error during merging for a rebase after I had tested it.  Guess I should retest :)